code/__DEFINES/preferences.dm

```diff
@@ -40,3 +40,4 @@
 #define BE_NINJA		1024
 #define BE_MONKEY		2048
 #define BE_GANG			4096
+#define BE_REVENANT		8192 //This is gonna conflict with shadowling :/

```

Comment:
With this, shadowling and abduction we're running out of bits. So some refactor will be needed.